routes/discord.js

```diff
@@ -18,4 +23,54 @@ router.post('/generate-invite', verifyRole(ROLE_NAMES.member), async (req, res)
   } 
 });
 
+router.get('/auth/discord', (req, res) => {
```

Comment:
same comment as in `oauth.js`, since we're planning more integrations, it would make sense to start grouping them now

Reply:
and I would even keep the discord-invite here and just have a separate file for auth endpoints. wdyt?

Reply:
Outdated.